kratos/containers/sparse_contiguous_row_graph.h

```diff
@@ -378,7 +379,7 @@ class SparseContiguousRowGraph
     ///@name Member Variables
     ///@{
     GraphType mGraph;
-    vector<LockObject> mLocks;
+    std::vector<LockObject> mLocks;
```

Comment:
to be honest i don't understand why this is a fix...

Reply:
btw i tried locally not to do the initialization in parallel (as you are implying here) and it was still segfaulting

Reply:
it seems like the ublas vector does some fishy stuff when resizing it
with std vector the intialization works properly, i.e. I think the default ctor is called which properly initializes the locks => then you don't need the initializatio in parallel